scripts/new-release.sh

```diff
@@ -36,7 +36,7 @@ elif [[ "$1" =~ $VERSION_REGEXP ]]; then
     if [ "$(git rev-parse --abbrev-ref HEAD)" == "$RELEASE_BRANCH_PREFIX$1" ]; then
         for dockerfile in $PATCH_DOCKERFILES; do
```

Comment:
`$PATCH_DOCKERFILES` contains only `Dockerfile-manager Dockerfile-manager.rhel7`, there should be probably all dockerfiles now